lib/popup_grid_view.dart

```diff
@@ -307,71 +296,143 @@ class PopupGridViewState extends State<PopupGridView> {
-                  ? (size.height - size.width) / 2
-                  : (size.height - size.width) / 6,
-              width: widget.side == DisplaySide.first? 0.0: size.width,
-            ),
-            Positioned(
-              bottom: widget.side == DisplaySide.second
-                  ? (size.height - size.width) / 4.2
-                  : null,
-              // top: widget.side == DisplaySide.second ? null : 200.0,
-              left: 0.0,
-              right: 0.0,
-              child: widget.side == DisplaySide.second
-                  ? Column(
-                      children: <Widget>[
-                        FittedBox(
-                          fit: BoxFit.fitHeight,
-                          child: new Row(
-                            mainAxisAlignment: MainAxisAlignment.center,
-                            children: width_val
-                                .map((widthValue) => Center(
-                                        child: RawMaterialButton(
-                                      onPressed: () {
-                                        model.painterController.thickness =
-                                            widthValue;
-                                        setState(() {
-                                          selectedWidth = widthValue;
-                                        });
-                                      },
-                                      constraints: new BoxConstraints.tightFor(
-                                        width: widthValue +
-                                            (size.height - size.width) * .02,
-                                        height: widthValue +
-                                            (size.height - size.width) * .02,
-                                      ),
-                                      fillColor: new Color(0xffffffff),
-                                      shape: new CircleBorder(
-                                        side: new BorderSide(
-                                          color: widthValue == selectedWidth
-                                              ? Colors.red
-                                              : Color(0xffffffff),
-                                          width: 2.0,
-                                        ),
-                                      ),
-                                    )))
-                                .toList(growable: false),
-                          ),
-                        ),
-                        SizedBox(
-                          height: (size.height - size.width) *.13,
-                          width: size.width ,
-                          child: ColorPicker(
-                              orientation: Orientation.portrait, model: model),
-                        ),
-                      ],
-                    )
-                  : Container(),
-            ),
+                color: Color(0xff283744),
```

Comment:
Have you tested if the old UI works properly?

Reply:
yes old UI working properly